theme/static/css/hydroshare_core.css

```diff
@@ -1473,12 +1473,15 @@ body > .container{
 
 #citation-text {
     display: inline-block;
-    float: left;
-	background:#EEE;
+    width:100%;
+    min-height: 30px;
+    background:#EEE;
+	border: 1px solid #cccccc;
+  	border-radius: 0px;
```

Comment:
@mauriel133 Good suggestion. Added padding rule and fixed alignment and spacing as well.